### DISCUSSION TITLE
Welcome to espn-nba-fantasy-analyzer Discussions!

### DISCUSSION THREAD

**Discussion by martincham** [martincham/espn-nba-fantasy-analyzer#2] [Announcements]
Hello! 

Post anything you have: question, thoughts, ideas that aren't exactly issues.

**Comment by Guccx** [comment]
my league has a different scoring where: oreb= 3 and dreb= 2 and other differences how can i edit this in the settings. Also reading the data is somewhat confusing os there a better explanation available

**Reply by Guccx** [reply]
in paticular what the different headings mean

**Reply by martincham** [reply]
1. There's no way to weight stats in the settings. You would have to go in and change the code.
2. Does the table at the very bottom of the README help explain the headings?
